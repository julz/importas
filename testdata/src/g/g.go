package g

import (
	v1 "knative.dev/serving/pkg/apis/serving/v1" // want `import "knative.dev/serving/pkg/apis/serving/v1" imported as "v1" but must be "kserving" according to config`
	knative1 "knative.dev/serving/pkg/queue"     // want `import "knative.dev/serving/pkg/queue" imported as "knative1" but must be "kqueue" according to config`
)

func foo() {
	v1.Resource(knative1.Name)
}
