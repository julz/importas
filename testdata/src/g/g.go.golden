package g

import (
	kserving "knative.dev/serving/pkg/apis/serving/v1" // want `import "knative.dev/serving/pkg/apis/serving/v1" imported as "v1" but must be "kserving" according to config`
	kqueue "knative.dev/serving/pkg/queue"             // want `import "knative.dev/serving/pkg/queue" imported as "knative1" but must be "kqueue" according to config`
)

func foo() {
	kserving.Resource(kqueue.Name)
}
